### DISCUSSION TITLE
Is the tabstospaces.py file supposed to convert tabs to spaces, or spaces to tabs?

### DISCUSSION THREAD

**Discussion by imeesa** [gri573/rethinking-voxels#121] [Q&A] [answered]
After some basic code review it appears like the latter, but the name does not seem to line up with that.

**Comment by gri573** [comment, marked as answer]
yes, it was supposed to do the latter, but it never really worked, so I just resorted to using vscodium's "convert indentation mode to tabs" command instead.

**Reply by imeesa** [reply]
Alright, thanks!
